packages/repo-tools/src/commands/knip-reports/knip-extractor.ts

```diff
@@ -170,15 +185,9 @@ export async function runKnipReports({
         ),
       ),
     );
+    await cleanKnipConfig({ knipConfigPath });
```

Comment:
should this be in a `finally` block so that it always happens?

Reply:
The way the command fails is that it spits out the invocation of `knip`, including the config file. If you then want to debug that, we can't really, unless the file is there.

That's why I changed this to leave the config file around when errors were hit.

Future scenario might even look like putting most of the config into a knip plugin, and then only the stuff specific to this instance of backstage like storyboard and internal packages would go into a knip config checked in to this repo. And other instances would do the same thing.

But we'll need a stable state for at least two backstages before that, I think.

Reply:
Hm OK. I see it's already in our gitignore - that's nice.